hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/action/Action.java

```diff
@@ -26,28 +25,17 @@
 import com.fasterxml.jackson.annotation.JsonInclude.Include;
 
 /**
- * A base class for action representation from the perspective of the alerts engine.
  * An action is the abstract concept of a consequence of an event.
- * A Trigger definition can be linked with a list of actions.
  *
- * Alert engine only needs to know an action id and message/payload.
- * Action payload can optionally have an event as payload.
- *
- * Action plugins will be responsible to process the action according its own plugin configuration.
+ * An Action object represents a particular action linked with a specific event.
+ * Action objects are generated by the Alerts engine and processed by plugins.
+ * An Action object stores the eventId property and optionally may contain the full Event object with its linked.
```

Comment:
Oh, I see what you are saying.  I think just leave it as:

"An Action object stores the eventId property and optionally may contain the full Event object."